app/pages/project/networking/VpcPage/VpcPage.tsx

```diff
@@ -49,16 +49,16 @@ export function VpcPage() {
           <PropertiesTable.Row label="DNS Name">{vpc.dnsName}</PropertiesTable.Row>
         </PropertiesTable>
         <PropertiesTable>
-          <PropertiesTable.Row label="Creation Date">
-            {vpc.timeCreated && formatDateTime(vpc.timeCreated)}
+          <PropertiesTable.Row label="Created">
```

Comment:
randomly noticed "Creation date" and thought that was weird. we only say it in a couple of spots